src/ozone/ozone.py

```diff
@@ -31,6 +31,16 @@
 RATE_LIMIT: int = 1
 
 
+def as_float(x: Any) -> float:
```

Comment:
Shouldn't this be a private method instead? `_as_float` instead of `as_float`

Reply:
Oh yeah, I forgot about that! 😄